backend/main/mutations/UserMutations.py

```diff
@@ -0,0 +1,55 @@
+import graphene
+
+from main.types.UserType import UserType
+from django.contrib.auth import get_user_model
+    
+class CreateUser(graphene.Mutation):
+    class Arguments:
+        username = graphene.String(required=True)
+        email = graphene.String(required=True)
+
+    user = graphene.Field(UserType)
+
+    def mutate(self, info, username, email):
+        user = get_user_model()(username=username, email=email)
```

Comment:
This is very confusing, but the way the `Graphene` docs describe how mutations should be called is not recommended. (The `Graphene-Django` docs are more explicit, see also this discussion if you're interested: https://github.com/graphql-python/graphene/issues/810.)

The heart of the matter is that `mutate` is not really a regular method, so if you inspect `self`, it will return as `None`. This means you can't use other class methods that you might have defined within the `CreateUser` class (which is silly). The Graphene-Django docs recommend marking the function as a `@classmethod`. The `cls` parameter then gives you access to other methods defined in this class. The argument `root` can be ignored (it refers to the 'parent' object, which is relevant for Types that are fields on other Types in the schema, but Mutations are top-level entities, so their root/parent is always `None`).

The suggested signature for this method (and mutatis mutandis also for the other mutations in this file) is:

```suggestion
from graphene import ResolveInfo

@classmethod
def mutate(cls, root: None, info: ResolveInfo, username: str, email: str):
```